language/move-vm/runtime/src/interpreter.rs

```diff
@@ -1349,6 +1352,39 @@ impl Frame {
                         gas_meter.charge_vec_swap(make_ty!(ty))?;
                         vec_ref.swap(idx1, idx2, ty)?;
                     }
+                    Bytecode::LdU16(int_const) => {
+                        gas_meter.charge_simple_instr(S::LdU16)?;
+                        interpreter.operand_stack.push(Value::u16(*int_const))?;
+                    }
+                    Bytecode::LdU32(int_const) => {
+                        gas_meter.charge_simple_instr(S::LdU32)?;
+                        interpreter.operand_stack.push(Value::u32(*int_const))?;
+                    }
+                    Bytecode::LdU256(int_const) => {
+                        gas_meter.charge_simple_instr(S::LdU256)?;
+                        interpreter.operand_stack.push(Value::u256(*int_const))?;
+                    }
+                    Bytecode::CastU16 => {
+                        gas_meter.charge_simple_instr(S::CastU16)?;
+                        let integer_value = interpreter.operand_stack.pop_as::<IntegerValue>()?;
+                        interpreter
+                            .operand_stack
+                            .push(Value::u16(integer_value.cast_u16()?))?;
+                    }
+                    Bytecode::CastU32 => {
+                        gas_meter.charge_simple_instr(S::CastU16)?;
+                        let integer_value = interpreter.operand_stack.pop_as::<IntegerValue>()?;
+                        interpreter
+                            .operand_stack
+                            .push(Value::u32(integer_value.cast_u32()?))?;
+                    }
+                    Bytecode::CastU256 => {
+                        gas_meter.charge_simple_instr(S::CastU16)?;
+                        let integer_value = interpreter.operand_stack.pop_as::<IntegerValue>()?;
+                        interpreter
+                            .operand_stack
+                            .push(Value::u256(integer_value.cast_u256()?))?;
+                    }
```

Comment:
Nit: can you please reorder these so that they appear next to the existing load and cast instructions?